asv_bench/benchmarks/groupby.py

```diff
@@ -29,7 +29,6 @@
         "skew",
         "cumprod",
         "cummax",
-        "rank",
```

Comment:
why are you removing this?

Reply:
That `dict` is for algos to not run for object types, so removed `rank` since it would now be supported